hive/lib/src/backend/stub/random_access_buffer.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:hive/hive.dart';
+import 'package:hive/src/binary/frame.dart';
+import 'package:hive/src/box/keystore.dart';
+
+abstract class RandomAccessBuffer {
```

Comment:
I think we don't need a separate implementation for the interface. This does not use any platform-specific APIs, in case anyone wants to re-implement it, they might just create a class implementing this. I'd recommend to unify `RandomAccessBuffer` and `RandomAccessBufferImpl` hence.

Reply:
Done!